index.css

```diff
@@ -133,6 +138,121 @@ header li a:hover {
   height: auto;
 }
 
+.page2 .title {
+  font-family: 'Crete Round', serif;
+}
+
+.page2 .buttontypeA {
+  margin-top: 10px;
+}
+
+.page2 .buttontypeA a {
+  text-decoration: none;
+  font-family: 'Inter', serif;
+  font-size: 17px;
+  color: white;
+  padding: 15px;
+  background-color: #ff6b00;
+}
+
+.page3 .buttontypeA a {
+  text-decoration: none;
+  font-family: 'Inter', serif;
+  font-size: 17px;
+  color: white;
+  padding: 15px;
+  background-color: #ff6b00;
+}
+
+.page2 .buttontypeB a {
+  display: block;
+  background: #ff6b00;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  text-decoration: none;
+  font-family: 'Inter', serif;
+  font-size: 20px;
+  color: white;
+}
+
+.page2 .buttontypeA a:hover {
+  background-color: #fff5e1;
+  border: 1px solid #ff6b00;
+  color: #ff6b00;
+}
+
+.page3 .buttontypeA {
+  margin-top: 10px;
+}
+
+.page3 .buttontypeA a:hover {
+  background-color: #fff5e1;
+  border: 1px solid #ff6b00;
+  color: #ff6b00;
+}
+
+.page2 .site1 {
+  height: 350px;
+  background-image: url(images/offenbach1.png);
+}
+
+.page2 .site2 {
+  background-image: url(images/spoiledeggs1.png);
+}
+
+.page2 .buttontypeB {
+  margin-top: 10px;
+  text-align: center;
+}
```

Comment:
- [ ] Please take care of margins on your "See project" button. It is separated from the rest of the card, but the original design does not have the button separated. 

Yours: 
![Captura de pantalla 2022-01-06 190218](https://user-images.githubusercontent.com/59100477/148429324-55c898db-f092-4b39-89cd-329f9157ddd7.jpg)

Figma template:
![Captura de pantalla 2022-01-06 190242](https://user-images.githubusercontent.com/59100477/148429356-65f02489-8b17-4039-9ea1-3e4fcb6c3f4f.jpg)